src/Microsoft.TemplateEngine.Edge/Settings/TemplateMatchInfo.cs

```diff
@@ -58,4 +62,39 @@ public void AddDisposition(MatchInfo newDisposition)
 
         public bool IsPartialMatch => MatchDisposition.Any(x => x.Kind != MatchKind.Mismatch);
     }
+
+    internal class TemplateMatchInfo2 : Abstractions.TemplateFiltering.ITemplateMatchInfo
```

Comment:
Do we have this type because we can't deprecate the other one otherwise? Are we going to rename this back to `TemplateMatchInfo` once the current `TemplateMatchInfo` is removed from the code base?

Reply:
exactly